src/main/resources/templates/pps/home.html

```diff
@@ -14,105 +14,97 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <h1 class="govuk-heading-xl" id="main-heading">
-                    Pay a penalty for filing your company accounts late
+                    Pay a penalty to Companies House
                 </h1>
-                <p class="govuk-body-l">Use this service to pay
-                    a company's late filing penalty.</p>
-
+                <p class="govuk-body-l">Use this service to make an online payment by credit or debit card if you've
+                    received a penalty notice from Companies House.</p>
                 <p class="govuk-body">You'll need:</p>
                 <ul class="govuk-list govuk-list--bullet" id="list-requirements">
-                    <li>the company number</li>
-                    <li>the reference number on the penalty notice</li>
-                    <li>to sign in to or create a Companies House account, which is different from a WebFiling account</li>
+                    <li>to sign in to or create a Companies House account (this is different to a WebFiling account)
+                    </li>
                     <li>a credit or debit card</li>
+                    <li>the information on your penalty notice (that we sent you in the post)</li>
                 </ul>
 
+                <div class="govuk-!-margin-top-6">
+                    <details class="govuk-details">
+                        <summary class="govuk-details__summary">
+                            <span class="govuk-details__summary-text">I've lost my penalty notice
+                            </span>
+                        </summary>
+                        <div class="govuk-details__text">
+                            <p class="govuk-body"> Email <a href="mailto:enquiries@companieshouse.gov.uk"
```

Comment:
This should be a mailto link e.g. `a href="mailto:enquiries@companieshouse.gov.uk"`
See `src/main/resources/templates/pps/confirmationPage.html`.
Also worth noting that their appears to be an an application property that we might be able to use in our template - see enquiries in `src/main/resources/application.properties`

Reply:
@rbull-ch please double check this as I have used a mailto link

Reply:
Ah sorry - right you are @klyttle-ch 
Was reviewing the prototype for first time and got my wires crossed!